forc-pkg/src/pkg.rs

```diff
@@ -2579,7 +2579,7 @@ pub fn check(
                 )
                 .unwrap(),
             );
-            lib_namespace_map.insert(node, namespace);
+            lib_namespace_map.insert(node, namespace.module().clone());
```

Comment:
Perhaps `lib_namespace_map` should be renamed here?

Reply:
I could, but I see "namespace" used as the variable name for `namespace::Module` throughout the compiler, so I think it's fine to keep. I'd rather not change this code more than necessary.